core/domain/prod_validation_jobs_one_off.py

```diff
@@ -820,7 +824,7 @@ def _validate_state_name(cls, item):
             # function.
             if exp_model is None or exp_model.deleted:
                 continue
-            if item.state_name not in exp_model.states.keys():
+            if item.state_name not in list(exp_model.states.keys()):
```

Comment:
You don't need to use keys, the `in` operator works by checking that for dicts.

```python
if item.state_name not in exp_model.states:
    # ...
```

Reply:
Done